tests/conftest.py

```diff
@@ -109,6 +109,37 @@ def DSHAPE(tmpdir_factory):
     return DSHAPE_out
 
 
+@pytest.fixture(scope="session")
```

Comment:
how is this different from the regular DSHAPE example thing? Do we need both? Or can they be combined?

Reply:
The one I added matches the input file in examples/DESC. The difference between the two inputs are the solver methods:
The one I added uses
```
# solver methods
optimizer         = lsq-exact
objective         = force
spectral_indexing = fringe
node_pattern      = jacobi
```
The one in tests uses
```
# solver methods
optimizer         = scipy-trf
objective         = force
spectral_indexing = ansi
node_pattern      = cheb2
```

- The tolerances I set for the mercier tests require using the first one to pass. The second one with `ansi` give a visible deviation in plots so the tolerance would need to be increased. I didn't want to do that since they were high anyway.
- I wanted to just replace the `cheb2` one with the `ansi` one, but a test in plotting relies on the `ansi` one (or at least failed on the `fringe`).

Reply:
hmm ok. using different indexing/grid should be fine, you might just need to regenerate some of the plot baseline images. The main reason for using the scipy optimizer was just to exercise that bit of the code. So as long as the coverage is still good im fine modifying the existing dshape test example to make it useful for you as well. I'm just trying to reduce the number of unique things we have to solve during testing since that tends to be the biggest bottleneck